Feature/post profile photo

bigyou98 opened the pull request:
PR

bigyou98 committed:
프로필 사진올리는 API 연동함
문제점
0. 새로고침을 하면 원상복귀됨
1. defult프로필 URL을 state의 초기값으로 연결해줘야함

bigyou98 committed:
현재 안되는 부분 새로고침시 이미지 원상복구

seungh0 commented on src/page/myPage/PhotoBox.jsx:
```diff
@@ -1,58 +1,96 @@
 
+export const PhotoBox = () => {
+  const [photo, setPhoto] = useState(
+    'https://potato-bucket-dev.s3.ap-northeast-2.amazonaws.com/7108d3d2-72e0-4f48-b2c1-86d2b8a41d22.png'
```
### 문제점
여기에  마이페이지 랜더링 될때 나의 회원정보를 불러오는 API를 호출하자나
그때 프로필 정보를 여기로 넘겨줘야돼

### 개선사항
추가적으로 URL 별도로 constant 폴더에
const DEFAULT_PROFILE = 'https://potato-bucket-dev.s3.ap-northeast-2.amazonaws.com/7108d3d2-72e0-4f48-b2c1-86d2b8a41d22.png'

이런식으로 분리해두면 사용하기 편할듯!

artvelop replied:
분리하는게 낫겠네요 ㄱㄱ

bigyou98 committed:
state 리덕스로 변경하는 대공사 하기전 개념 정리
* constant로 기본프로필사진 url 저장해놨다.

bigyou98 committed:
사진업로드 기능은 구현이 됬지만 좀더 세부적인 기능에 대한 문제가 있다.
1. 수정버튼을 눌러야 프로필이미지도 수정됨
2. 처음 페이지 불러올때아직 이미지 등록전에 이미지가  아무것도 안떠있던 것 같음 지금 나는 이미 수정해서 맨 처음의 것을 확인못하지만 기본이미지로 등록이 안되있던것 같았다.
